### DISCUSSION TITLE
Poor transcription

### DISCUSSION THREAD

**Discussion by virtualsapiens** [deepgram/community#569] [General help] [answered]
### Which Deepgram product are you using?

Deepgram API

### Details

I have a .wav file that for some reason misses a good 25% of the transcription.  Doesn't matter if I use API, Playground, etc.  The audio is very clear when playing back, no background noise, etc.  I'd like to discuss this w/ Deepgram support but don't want to publicly post the audio.  How can I go about this?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Reply by virtualsapiens** [reply]
My request_id (testing it in playground) was dad02086-d4f8-482a-ba5b-655de4f8bff4

**Comment by jpvajda** [comment, marked as answer]
@virtualsapiens  you might want to give us a try again, since this post (we apologize for not responding sooner) we've releases our [Nova-2 ](https://developers.deepgram.com/docs/models-languages-overview#nova-2)model which might be a much better experience for you. 



If you try again and have similar issues, just let us know by creating a new Discussion or joining our [Discord Server](https://dpgr.am/discord) where our Community can try to assist you.
